config/schema/rules.schema.yml

```diff
@@ -29,6 +29,38 @@ rules.component.*:
       type: rules_expression.[%parent.expression_id]
       label: 'Configuration'
 
+rules.reaction.*:
+  type: config_entity
+  label: 'Reaction rule'
+  mapping:
+    id:
+      type: string
+      label: 'ID'
+    label:
+      type: label
+      label: 'Label'
+    event:
+      type: string
+      label: 'Event'
+    module:
+      type: string
+      label: 'Module'
+    description:
+      type: text
+      label: 'Administrative description'
+    tag:
+      type: string
+      label: 'Tag'
+    core:
+      type: string
+      label: 'Drupal version'
```

Comment:
That is basically a copy of View.php from core, which also saves the core compatibility flag. Not sure we need it, but I would keep it for now.